tests/torch/test_sota_checkpoints.py

```diff
@@ -9,684 +9,535 @@
-                    str(diff_target),
-                    str("-"),
-                ]
-            else:
-                row = [str(key), str(metrics_type_), str(expected_), str(metric), str(diff_target), str("-")]
-        else:
-            if test == "eval":
-                row = [
-                    str(key),
-                    str(metrics_type_),
-                    str(expected_),
-                    str("Not executed"),
-                    str(fp32_metric_),
-                    str("-"),
-                    str("-"),
-                    str(error_message),
-                ]
-            else:
-                row = [str(key), str(metrics_type_), str(expected_), str("Not executed"), str("-"), str(error_message)]
-        return row
+    def read_metric(metric_file_name: str):
+        with open(metric_file_name, encoding="utf8") as metric_file:
+            metrics = json.load(metric_file)
+        return metrics["Accuracy"]
 
     @staticmethod
-    def write_error_in_csv(error_message, filename):
-        with open(f"{filename}.csv", "w", newline="", encoding="utf8") as csvfile:
-            fieldnames = [
-                "model",
-                "launcher",
-                "device",
-                "dataset",
-                "tags",
-                "metric_name",
-                "metric_type",
-                "metric_value",
-            ]
-            writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
-            writer.writeheader()
-            writer.writerow(
-                {
-                    "model": filename,
-                    "launcher": "-",
-                    "device": "-",
-                    "dataset": "-",
-                    "tags": "-",
-                    "metric_name": "-",
-                    "metric_type": "-",
-                    "metric_value": error_message,
-                }
-            )
+    def generate_accuracy_check_cmd(config_path: Path, ov_data_dir: str, model_folder: Path, report_csv_path: Path):
+        cmd = [
+            "accuracy_check",
+            "--config", config_path.as_posix(),
+            "--source", ov_data_dir,
+            "--definitions", DATASET_DEFINITIONS_PATH.as_posix(),
+            "--models", model_folder.as_posix(),
+            "--csv_result", report_csv_path.as_posix(),
+        ]  # fmt: skip
+        return " ".join(cmd)
+
+    def get_reference_fp32_metric(self, metrics_dump_path: Path, reference_name: str) -> Tuple[Optional[float], bool]:
+        fp32_metric = None
+        if reference_name is not None:
+            fp32_metric = self.ref_fp32_dict[reference_name]
+            reference_metric_file_path = self.get_metric_file_name(metrics_dump_path, reference_name)
+            if reference_metric_file_path.exists():
+                acc = self.read_metric(reference_metric_file_path)
+                if acc:
+                    fp32_metric = acc
 
-    def write_results_table(self, init_table_string, path):
```

Comment:
Look on `results.csv` file, that contains description of errors in new "Status" column. 
Html report available on Jenkins.

Reply:
Ok, so it seems that the visualization code is now run from the CI pipeline. I don't really like this since this takes the visualization out of our hands and we cannot improve it if necessary. The popular vote was for the new visualization format, so I won't block on this.